src/components/staking/components/StakingAction.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react'
+import AmountInput from './AmountInput'
+import ValidatorBox from './ValidatorBox'
+import FormButton from '../../common/FormButton'
+import { Translate } from 'react-localize-redux'
+import ArrowCircleIcon from '../../svg/ArrowCircleIcon'
+import TransferMoneyIcon from '../../svg/TransferMoneyIcon'
+import StakeConfirmModal from './StakeConfirmModal'
+import BN from 'bn.js'
+import { utils } from 'near-api-js'
+import isDecimalString from '../../../utils/isDecimalString'
+import { onKeyDown } from '../../../hooks/eventListeners'
+import { toNear } from '../../../utils/amounts'
+
+const {
+    parseNearAmount, formatNearAmount
+} = utils.format
+
+export default function StakingAction({
+    match,
+    validators,
+    loading,
+    availableBalance,
+    hasLedger,
+    action,
+    handleStakingAction
+}) {
+    const [confirm, setConfirm] = useState()
+    const [amount, setAmount] = useState('')
+    const [success, setSuccess] = useState()
+    const [useMax, setUseMax] = useState(null)
+    const validator = validators.filter(validator => validator.accountId === match.params.validator)[0]
+    const hasStakeActionAmount = !loading && amount.length && amount !== '0'
+    let staked = (validator && validator.staked) || '0'
+    const stake = action === 'stake' ? true : false
+    const displayAmount = useMax ? formatNearAmount(amount, 5) : amount
+    const decimalDeviation = '0.00001'
```

Comment:
this could be a const outside of StakingAction function (module level after imports) and value could be `new BN(parseNearAmount('0.00001'))` then replace usage with DECIMAL_DEVIATION